### PR TITLE
Smart pistol Rework. Everyone can now use it.

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Renames Smart pistol to Smart machinepistol to better fit the theme.
Everyone can now use the smart machinepistol. It has been removed from SG vendors as default.
As it is now universally usable by anyone it now costs 250 points rather than 100.
Resprite of the smart machinepistol done by me.
I also added some SG essential kits since I moved the pistol out of the SG kit so robots don't get randomly trolled out of their NVDs.

Stats below if you really care.

image

<details>

---

**Name:** SP-13 Smart machinepistol [9mm AP]
**Damage Profiles:** 20/12.5/0.5 [Pistol AP]
**Fire Delay/Burst:** 0.2 Fire Delay, N/A Burst. [T-12 level for reference.]
**Gun Handling:** .35 Wielded Slowdown and .6 Wield Delay. [Handling is mildly worse]
**Capacity:** 40 [Changed to be higher from 30.]
**Attachments**: Removed barrel and underbarreled attachments. Added magnetic harness.
**Scatter/Accuracy**: Improved base unwielded scatter.

**Overall:** This works as a IFF secondary. It is a pretty bad SMG on its own. It could maybe have the capacity buff removed if it shows to be too strong.

</details>

## Why It's Good For The Game
Reworks the smartpistol from its weirdo state where only SGs can use it as their weird smg tritary and reworks it to be closer to a IFF sidearm people can shell out for. Its still cheaper than a scout rifle but at 325 it should be inticing for people to pick up.
## Changelog
:cl:
balance: The Smart machinepistol has been redesigned from being an SG sidearm to being a proper IFF sidearm. It no longer requires SG skill to use and no longer has underbarreled or barrel attachments. It has mildly worse handling than before and better base scatter. It now costs 325 in req to reflect the fact it is now usable by anyone.
balance: SGs no longer acquire Smartpistols for free.
tweak: Smart pistol is now called Smart machinepistol to better reflect its role.
/:cl:

### CODE REVIEW

**Comment by FosterGrime-Heymynameslouie** [comment]
honestly always assumed that anyone can use the smart pistol when it was first conceived and that it would be far more useful and fun that way 

pretty cool pr spedmrhg

**Comment by ilikesaying** [comment]
image
very cool SmartMachinePistol sprite

File: code/modules/projectiles/guns/pistols.dm [hunk 1]
```diff
@@ -833,43 +833,37 @@ It is a modified Beretta 93R, and can fire three round burst or single fire. Whe
...
 	icon_state = "sp13"
 	worn_icon_state = "sp13"
 	caliber = CALIBER_9X19 //codex
-	max_shells = 30 //codex
-	fire_sound = 'sound/weapons/guns/fire/tp14.ogg' //same bullets, same sound
+	max_shells = 40 //codex
+	fire_sound = 'sound/weapons/guns/fire/tp14.ogg' //
 	reload_sound = 'sound/weapons/guns/interact/tp14_reload.ogg'
 	default_ammo_type = /obj/item/ammo_magazine/pistol/standard_pistol/smart_pistol
 	allowed_ammo_types = list(/obj/item/ammo_magazine/pistol/standard_pistol/smart_pistol)
```
Comment (Helg2) [inline code/modules/projectiles/guns/pistols.dm:847]: rename it atleast

**Comment by Lauwma** [comment]
325 points, inflation is real

**Comment by iforgotmeword** [comment]
as long as you can dual weld/(or at least 1 hand them) them im happen with the change

**Comment by SimplySmiley123** [comment]
325 seems like a _lot_ of points for a glorified mp19 with no attachments.

**Comment by Pariah919** [comment]
> 325 seems like a _lot_ of points for a glorified mp19 with no attachments.

Realistically this is an IFF gun that fits in your belt slot. While scout rifles are only... what? 75 more? Points this is still pretty strong considering that.
I could reduce it to around 225~ and see how that plays out instead.

**Comment by SimplySmiley123** [comment]
> Realistically this is an IFF gun that fits in your belt slot. While scout rifles are only... what? 75 more? Points this is still pretty strong considering that. I could reduce it to around 225~ and see how that plays out instead.

Yeah, pretty much.
Why get an SMG that already kinda exists just without the IFF and free from the get-go, if you could get something that packs more mean of a punch at longer ranges for a radiopack and a half more worth of points.

**Review by Kuro020 — APPROVED** [review]

**Comment by Pariah919** [comment]
Swapped to 250 points.

**Review by TiviPlus — APPROVED** [review]

**Comment by DrAmazing343** [comment]
Having played with this a bit, I'll say that it feels clunky to use as a main weapon at all. It'd be highly appreciated if you could put magazines in the weapons vendor since the actual ammo, 9mm AP, is available via packets in the Boxes tab already. It's fun to use, being IFF and decently fire-rated, but with only two magazines in my webbing and one in the gun, I find myself spending far, far more time reloading the damn thing than anything else. At least, the one that comes with the SG kit only gets two spare mags and it's annoying to obtain more.

Otherwise, even being IFF-capable, I'm glad you dropped the points for it to 250. Feels a lot more in-line with what it's worth given the damage profile/falloff and lack of attachment slots. The sprite's also badass, and half the reason I chose to actually use it in the first place. Well done!

**Comment by Breadeagle** [comment]
Messed around with it and it feels kind of bad. Can't one hand it like most pistols since it lacks attachments, ammo type is pretty weak for what the guns stats are. Despite using the mk88 ammo type, you can't vend unlimited mags so you have to manually reload every mag - its just not worth it. Complete resource sink at any req as is, if you intend on using it as a primary, which funnily enough, most pistols can do right now with infinite mags and solid stats.

The IFF is something, but if we're at a point you're getting a gun from req at all, the BR-8 is right there, and can even have better handling due to its attachment options. Underbarrel options, better ammo, etc. I'm aware the prices are different, but with the smart pistol you're arguably paying for a down grade in many cases, where paying for an upgrade you should consider the value how much it improves your previous option.

**Pull request merged**